presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -117,15 +117,15 @@ public QueryManagerConfig setInitialHashPartitions(int initialHashPartitions)
     }
 
     @NotNull
-    public Duration getMaxQueryAge()
+    public Duration getMinQueryExpireAge()
     {
-        return maxQueryAge;
+        return minQueryExpireAge;
     }
 
-    @Config("query.max-age")
-    public QueryManagerConfig setMaxQueryAge(Duration maxQueryAge)
+    @Config("query.min-expire-age")
```

Comment:
I think this is change requested earlier by @martint already introduced in fixup commit.